zebra-chain/src/work/difficulty.rs

```diff
@@ -256,7 +256,27 @@ impl ExpandedDifficulty {
     /// Hashes are not used to calculate the difficulties of future blocks, so
     /// users of this module should avoid converting hashes into difficulties.
     fn from_hash(hash: &block::Hash) -> ExpandedDifficulty {
-        ExpandedDifficulty(U256::from_little_endian(&hash.0))
+        U256::from_little_endian(&hash.0).into()
+    }
+
+    /// Returns the easiest target difficulty allowed on `network`.
+    ///
+    /// See `PoWLimit` in the Zcash specification.
+    pub fn target_difficulty_limit(network: Network) -> ExpandedDifficulty {
+        let limit: U256 = match network {
+            /* 2^243 - 1 */
+            Network::Mainnet => (U256::one() << 243) - 1,
+            /* 2^251 - 1 */
+            Network::Testnet => (U256::one() << 251) - 1,
+        };
+
+        limit.into()
+    }
+}
```

Comment:
might want to re-rexport this as a method on `Network`

Reply:
I think we'll only use this method twice in the difficulty module: once for the PoWLimit check, and once for the minimum difficulty during difficulty adjustment.

So I'm not sure if it needs to go in a method - maybe as part of #1135?